nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-client-service/pom.xml

```diff
@@ -204,181 +215,42 @@
             <activation>
                 <activeByDefault>false</activeByDefault>
             </activation>
+            <properties>
+                <elastic_docker_image>7.17.6</elastic_docker_image>
```

Comment:
Personally I prefer being specific "Elastic" is a company name, `elasticsearch` is the application being used ... but I'm not going to insist on a change for the PR

Reply:
Done.